jjb/mkck/mkck-trigger.yaml

```diff
@@ -593,6 +624,140 @@
             include-test-summary: true
             build-server: 'http://ci.ses.suse.de:8080/'
 
+- job-template:
```

Comment:
this has to much of code duplication, can you try and use only custom builders, like it is done jjb/storage-ci.yaml or you've used macros:

```
- builder:
    name: copy_artifacts_opensuse
    builders:
        - copyartifact:
        . . .
- builder:
    name: copy_artifacts_sles
    builders:
        . . .    
.
.
.

builders:
    - copy_artifacts_{dist}
.
.
.
```

Reply:
I don't think that will work. In the SLES case, we have multiple steps

```
- multijob:
            name: Deliver artifacts
- copyartifact:
            project: storage-deliver-mkck-artifact 
- copyartifact:
            project: storage-deliver-update-repos
- shell: |
            . v/bin/activate
            . sesci/common/util
            for i in artifacts*.yaml ; do

..... something else here

- shell: |
            . v/bin/activate
            . sesci/common/util
            for i in artifacts*.yaml ; do
```

For openSUSE, we only have a single step. Using macros would mean, that I need noop macros for some of the steps for openSUSE.
Also the order is different - `deliver artifacts` for SLES runs on the jenkins slave (compute), but adding the repos on openSUSE runs on the dynamically created jenkins slave (on OVH).

Reply:
I created another PR with the approach with noop macros. See https://github.com/SUSE/sesci/pull/31